dev/fattest.simplicity/src/com/ibm/websphere/simplicity/RemoteFile.java

```diff
@@ -110,6 +110,9 @@ public boolean retry(Operation op, long retryNs) throws Exception {
         }
 
         for ( int retryNo = 0; !didAct && retryNo < retryCount; retryNo++ ) {
+            if (retryNo != 0) {
```

Comment:
remove this guard ... want to sleep before all retries

Reply:
Correct, I didn't realise it did an initial attempt outside of the loop.

Reply:
Now fixed